packages/vscode-extension/README.md

```diff
@@ -1,64 +1,85 @@
-- [Import Objects]
-- [Import Files]
-- [Create Project]
-- [Create Files]
+To set an account as default for a project, follow these steps:
+1. Click Ctrl+Shift+P to open the Command Palette.
+2. Enter **SuiteCloud** and from the dropdown list, select **SuiteCloud: Set Up Account**.
+3. From the dropdown list, select the authentication ID (authID) of the account you want to set as default.
+The selected account is set as default for the project.
 
-💡 To read more about what you can do with SuiteCloud extension for Visual Studio Code, visit [NetSuite's Help Center](https://system.netsuite.com/app/help/helpcenter.nl?fid=article_159223173518.html).
+<p align="center">
+  <img src="resources/SetUpAccount.gif" alt="animated" />
+</p>
 
-## SuiteCloud Extension combined with SuiteCloud CLI for Node.js
-SuiteCloud CLI for Node.js is another open-source SuiteCloud SDK tool. This tool has more available commands that you can use in combination with the ones available in SuiteCloud extension for Visual Studio Code.
+### <font size="5"> <span style="color:gray"> SuiteCloud Commands </font> </span>  
 
-To work with both tools at the same time, do the following:
-1. Install SuiteCloud CLI for Node.js by following the instructions [here](../node-cli/README.md).
-2. Open a terminal inside of your SDF project, and enter:
-   ```javascript
-   suitecloud -h
-   ```
-3. You can now use SuiteCloud CLI for Node.js in your SDF projects while working in SuiteCloud extension for Visual Studio Code.
+The following SuiteCloud commands are available through the Command Palette:
 
-![SuiteCloud CLI for Node.js combined with SuiteCloud extension for Visual Studio Code](resources/cliForNodejsDemo.gif)
+| Command | Description | Shortcut |
+| --- | --- | --- |
+|<a href="https://system.netsuite.com/app/help/helpcenter.nl?fid=section_160147339580.html" rel="noopener noreferrer" target=blank>Add Dependency References to the Manifest</a>|Adds the defined dependencies to the Manifest file.| Ctrl+Shift+Alt+A |
+|<a href="https://system.netsuite.com/app/help/helpcenter.nl?fid=section_160147328227.html" rel="noopener noreferrer" target=blank>Create SuiteScript Files</a>|Creates a SuiteScript file based on a script type template that also includes any modules you select to extend NetSuite and customize, search for, and process your data. 
+|<a href="https://system.netsuite.com/app/help/helpcenter.nl?fid=section_162938506015.html" rel="noopener noreferrer" target=blank>Create Project</a>|Creates SDF file-based projects to develop ACPs or SuiteApp projects for internal use or for commercial distribution.|
+|<a href="https://system.netsuite.com/app/help/helpcenter.nl?fid=section_160147342366.html" rel="noopener noreferrer" target=blank>Deploy Project</a>|Generates a deploy.xml file that deploys all supported files in the **FileCabinet** folder and all custom NetSuite objects in the **Objects** folder. | Ctrl+Alt+D |
```

Comment:
Does it Generate the deploy.xml file ??
From nodecli:
Deploys the folder containing the project. The project folder is zipped before
                                deployment, only including the files and folders referenced in the deploy.xml
                                file.

Reply:
My bad, I misunderstood. Deleting it and using: _Deploys the folder containing the project. The project folder is zipped before deployment including only the files and folders referenced in the deploy.xml file._